capella2polarion/converters/document_config.py

```diff
@@ -1,15 +1,16 @@
 # Copyright DB InfraGO AG and contributors
 # SPDX-License-Identifier: Apache-2.0
 """Module with classes and a loader for document rendering configs."""
+import collections.abc as cabc
 import logging
 import pathlib
 import typing as t
 
 import capellambse
 import jinja2
-import polarion_rest_api_client as polarion_api
 import pydantic
 import yaml
+from polarion_rest_api_client import data_models
```

Comment:
Why do we do this? I would prefer to keep it as is to highlight that we use things from the polarion_rest_api_client

Reply:
How is this not highlighted, if we import a package from it? This is also a change respecting our style guide. We can rename the imported package:

```suggestion
from polarion_rest_api_client import data_models as polarion_api_data_models
```

Reply:
The main issue is that we also have a module named `data_models` in this project. But also from the conceptual perspective, I prefer to import things from a higher level, if they are provided on higher level. Let's assume we decide that the `data_models` module in polarion_rest_api_client has grown too much and should be split. With this approach, that would be a breaking change. If we import the polarion_rest_api_client only and use classes from there, the underlying module structure could be changed without any issues

Reply:
Good point with the overlapping names. That points me to rename the imported submodule to `polarion_api_data_models`. 

I try to understand you: You say that all needed objects are "exported" in the `__init__` of the polarion_rest_api_client? That isn't the case for `Layouter`. So this is an issue there then.